tools/scoring/score.py

```diff
@@ -463,10 +464,14 @@ def score(division,
                 #       reports unsat.  We check for particular error messages
                 #       that indicate crashes, timeouts, or sat without
                 #       any model ("sat expected" or "( expected").
+                #       Due to some unknown problem the jobs on fixed and
+                #       best of 2019 solvers produced no entries ("-") in
+                #       that column.
                 incomplete_model = data_new['model_validator_exception']\
```

Comment:
For completeness: there is no such column in the StarExec Job_info files.  The "-" is added by a script in scoring.